tests/test_inject.py

```diff
@@ -47,3 +49,19 @@ def test_inject_include_apps(pipx_temp_env, capsys, with_suffix):
         assert run_pipx_cli(["inject", "pycowsay", PKG["black"]["spec"], "--include-deps"])
 
     assert not run_pipx_cli(["inject", f"pycowsay{suffix}", PKG["black"]["spec"], "--include-deps"])
+
+
+def test_inject_with_req_file(pipx_temp_env, capsys, tmp_path):
+    req_file = tmp_path / "requirements.txt"
```

Comment:
Since that is, as you mentioned, not a regular `requirements.txt` file, it's probably better to modify is slightly?

```suggestion
    req_file = tmp_path / "inject-requirements.txt"
```